src/charts/utilization-bar/utilization-bar-chart.directive.js

```diff
@@ -0,0 +1,153 @@
+       <div pf-utilization-bar-chart chart-data=data1 chart-title=title1 units=units1></div>
+       <hr>
+       <div pf-card head-title="Utilization Bar Chart">
+         <div pf-utilization-bar-chart chart-data=data2 chart-title=title2 units=units2 thresholds=thresholds></div>
+         <div pf-utilization-bar-chart chart-data=data3 chart-title=title3 units=units3 thresholds=thresholds></div>
+         <div pf-utilization-bar-chart chart-data=data4 chart-title=title4 units=units4 thresholds=thresholds></div>
+         <div pf-utilization-bar-chart chart-data=data5 chart-title=title5 units=units5 thresholds=thresholds></div>
+       </div>
+
+       <hr>
+       <label><strong>layout='inline', used-label-format='percent'</strong></label>
+       <div pf-card head-title="Utilization Bar Chart">
+         <div pf-utilization-bar-chart chart-data=data2 chart-title=title2 layout=layoutInline used-label-format='percent' units=units2 thresholds=thresholds></div>
+         <div pf-utilization-bar-chart chart-data=data3 chart-title=title3 layout=layoutInline used-label-format='percent' units=units3 thresholds=thresholds></div>
+         <div pf-utilization-bar-chart chart-data=data4 chart-title=title4 layout=layoutInline used-label-format='percent' units=units4 thresholds=thresholds></div>
+         <div pf-utilization-bar-chart chart-data=data5 chart-title=title5 layout=layoutInline used-label-format='percent' units=units5 thresholds=thresholds></div>
+       </div>
+     </div>
+   </file>
+
+   <file name="script.js">
+   angular.module( 'patternfly.example', ['patternfly.charts', 'patternfly.card']);
+
+   angular.module( 'patternfly.example' ).controller( 'ChartCtrl', function( $scope ) {
+
+    $scope.thresholds = {'warning':'60','error':'85'};
+
+    $scope.title1 = 'RAM Usage';
+    $scope.units1 = 'MB';
+
+    $scope.data1 = {
+      'used': '8',
+      'total': '24'
+    };
+
+    $scope.title2 = 'Memory Utilization';
+    $scope.units2 = 'GB';
+
+    $scope.data2 = {
+      'used': '25',
+      'total': '100'
+    };
+
+    $scope.title3 = 'CPU Usage';
+    $scope.units3 = 'MHz';
+
+    $scope.data3 = {
+      'used': '420',
+      'total': '500',
+    };
+
+    $scope.title4 = 'Disk Usage';
+    $scope.units4 = 'TB';
+    $scope.data4 = {
+      'used': '350',
+      'total': '500',
+    };
+
+    $scope.title5 = 'Disk I/O';
+    $scope.units5 = 'I/Ops';
+    $scope.data5 = {
+      'used': '450',
+      'total': '500',
+    };
+
+    $scope.layoutInline = {
+      'type': 'inline',
+      'titleLabelWidth': '120px',
+      'usedLabelWidth': '60px'
+    };
+   });
+   </file>
+ </example>
+*/
+
+angular.module('patternfly.charts').directive('pfUtilizationBarChart', function ($timeout) {
+  'use strict';
+  return {
+    restrict: 'A',
+    scope: {
+      chartData: '=',
+      chartTitle: '=',
+      units: '=',
+      thresholds: '=?',
+      usedLabelFormat: '@?',
+      layout: '=?'
+    },
+    replace: true,
```

Comment:
This is [deprecated](https://docs.angularjs.org/api/ng/service/$compile#-replace-deprecated-will-be-removed-in-next-major-release-i-e-v2-0-).